src/main/java/frc/bionic/TrajectoryUtils.java

```diff
@@ -0,0 +1,49 @@
+package frc.bionic;
+
+import java.util.Arrays;
+
+import edu.wpi.first.wpilibj.geometry.Pose2d;
+import edu.wpi.first.wpilibj.geometry.Rotation2d;
+import edu.wpi.first.wpilibj.geometry.Translation2d;
+import edu.wpi.first.wpilibj.trajectory.Trajectory;
+import edu.wpi.first.wpilibj.trajectory.TrajectoryConfig;
+import edu.wpi.first.wpilibj.trajectory.TrajectoryGenerator;
+import frc.bionic.swerve.AbstractDrivetrain;
+
+public class TrajectoryUtils {
+
+      /**
+       * 
+       * @param startPos The starting position of the robot on the trajectory
+       * @param endPos The ending position og the robot on the trajectory
+       * @param config The Trajectory config object that defines max velocity and max acceleration
+       * @param interiorWaypoints An Arraylist of Translation 2d object which generate the path in between the start and end
+       * @return The generated trajectory based off of the given parameters
+       */
+      public static Trajectory generateTrajectory(Pose2d startPos, 
+                                            double endX,
+                                            double endY,
+                                            double rotation, 
+                                            AbstractDrivetrain drivetrain, 
+                                            Translation2d... interiorWaypoints) {
+      // Creates a new Trajectory Config, which stores Max Velocity, Max Acceleration and Kinematics
+      TrajectoryConfig config =
+            new TrajectoryConfig(
+                    // Max Velocity Meters Per Seconds
+                    Conversion.inchesToMeters(120),
+                    // Max Acceleration Meters Per Seconds Squared
+                    Conversion.inchesToMeters(144))
+                // Add kinematics to ensure max speed is actually obeyed
+                .setKinematics(drivetrain.getKinematics());
+          //Updates Shuffleboard
+
+      Pose2d endPos = new Pose2d(endX, endY, Rotation2d.fromDegrees(rotation));
+      Trajectory returnTrajectory = TrajectoryGenerator.generateTrajectory(startPos, 
+                                                                            Arrays.asList(interiorWaypoints), 
+                                                                            endPos, 
+                                                                            config);
+      return returnTrajectory;/*.transformBy(
+                                      new Transform2d(drivetrain.getCurrentPose().getTranslation(), 
+                                      Rotation2d.fromDegrees(0))); */
+      }
```

Comment:
If the `transformBy` stuff has been shown to be unnecessary, shouldn't it be deleted now?